beebop/resources/args.json

```diff
@@ -16,7 +16,9 @@
         "save_partial_query_graph": true,
         "min_kmer_count": 0,
         "exact_count": false,
-        "serial": false
+        "serial": false,
+        "stable": null,
+        "use_full_network": false
```

Comment:
Are these new properties required by poppunk, or just related to this particular version?